mmpose/models/detectors/multiview_pose.py

```diff
@@ -53,7 +53,7 @@ def compute_grid(self, box_size, box_center, num_bins, device=None):
             grid_1D_x + box_center[0],
             grid_1D_y + box_center[1],
             grid_1D_z + box_center[2],
-        )
+            indexing='ij')
         grid_x = grid_x.contiguous().view(-1, 1)
```

Comment:
The argument `indexing` has been introduced in PyTorch1.10.0 so we need to handle the compatibility of earlier versions. I think a version check could be added like:

```python
from packaging import version

if version.parse(torch.__verson__) < version.parse('1.10.0'):
    grid_x, grid_y, grid_z = torch.meshgrid(
            grid_1D_x + box_center[0],
            grid_1D_y + box_center[1],
            grid_1D_z + box_center[2])
else:
    grid_x, grid_y, grid_z = torch.meshgrid(
            grid_1D_x + box_center[0],
            grid_1D_y + box_center[1],
            grid_1D_z + box_center[2],
            indexing='ij')
```

Reply:
Yes, or seeing as there are multiple instances of `torch.meshgrid` one could write a wrapper for it that does the version check. Where would be an appropriate location for such a wrapper?

Reply:
I think it's okay to directly use an if-else statement. If you prefer to use a wrapper, what about placing it at `mmpose/models/utils/misc.py`? Also, please note that `mmpose/models/backbones/utils/transformer.py` has been moved to 'mmpose/models/utils/transformer.py` this PR may need a rebase.

Reply:
I have added a wrapper and adjusted